docs/pages/what-are-virtual-clusters.mdx

```diff
@@ -46,3 +46,30 @@ Finally, virtual clusters can be configured independently of the physical cluste
     <figcaption>vCluster - Comparison</figcaption>
 </figure>
 
+## Benefits of Virtual Clusters
+
+Virtual clusters provide immense benefits for large-scale Kubernetes deployments and multi-tenancy:
+- **Full Admin Access**:
+    - Deploy operators with CRDs, create namespaces and other cluster scoped resources that's usually not possible in namespaces.
+    - Taint and label nodes without any influence on the host cluster.
+    - Reuse and share services across multiple virtual clusters with ease.
+- **Cost Savings:**
+    - You can create lightweight vClusters that share the underlying host cluster instead of creating separate "real" clusters.
+    - vClusters are just deployments, so they can be easily auto-scaled, purged, snapshotted and moved.
+- **Low Overhead:**
+    - vClusters are super lightweight and only reside in a single namespace.
+    - vClusters run with K3s, a super low-footprint K8s distribution, but they can also run with "real" K8s.
+    - The control plane of a vCluster runs inside a single pod (+1 CoreDNS pod for vCluster-internal DNS capabilities).
+- **<u>No</u> Network Degradation:**
+    - Since the pods and services inside a vCluster are actually being synchronized down to the host cluster\*, they are effectively using the underlying cluster's pod and service networking and are therefore not a bit slower than any other pods in the underlying host cluster.
+- **API Server Compatibility:**
+    - vClusters run with the K3s API server which is certified K8s distro which ensures 100% Kubernetes API server compliance.
+    - vCluster have their own API server, controller-manager and a separate, isolated data store (sqlite for easiest option but this is configurable, you can also deploy a full-blown etcd if needed).
```

Comment:
It could be more precise here. Does a vCluster run with the K3s API server or with its own?

Maybe something like "vCluster manages its API server" would be more explicit.